commands/download_mta_op_logs_command_test.go

```diff
@@ -49,15 +49,15 @@ var _ = Describe("DownloadMtaOperationLogsCommand", func() {
 					processId, org, space, user),
 				"OK\n",
 				fmt.Sprintf("Saving logs to %s"+string(os.PathSeparator)+"%s...\n", wd, dir),
-				fmt.Sprintf("  %s\n", testutil.LogID),
+				fmt.Sprintf("  %s.log\n", testutil.LogID),
```

Comment:
Not consistent with other concatenations.
fmt.Sprintf("  %s\n", testutil.LogID + ".log") -> Looks better to me.
Consider extracting these concatenations in a new method. Something like addLogExtension...